content/ja/helpdesk.md

```diff
@@ -5,7 +5,7 @@ publishdate: 2012-01-10T00:05:19.000Z
 draft: false
 title: ヘルプデスク
 slug: helpdesk
-subtitle: ３人寄れば文殊のサポート
+subtitle: バイリンガルのITエンジニアがシステム不具合をサポート
 description: イソリア - 様々なトラブルや解決策を集結させた社内データベースを利用し、迅速で信頼性の高いサポートを提供
```

Comment:
@yasuyoshiakiyama these seem only partially done yoshi. Description should have the key phrase.